frontend/src/components/ViewTrips/save-trip-form-elements.js

```diff
@@ -43,13 +43,13 @@ function createDateFormControl(ref, defaultDate) {
 }
 
 /**
- * Returns a Form.Control element with input type 'email' and other props
- * specified by the function parameters.
+ * Returns a Form.Control element with input type `email` and specified values
```

Comment:
Should Form.Control have backticks surrounding it like the previous comments?

Reply:
Yes. Added to all instances.